entities/params/postseskeys.go

```diff
@@ -0,0 +1,39 @@
+package params
+
+import (
+	"strings"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/sirupsen/logrus"
+
+	myvalidator "github.com/mailbadger/app/validator"
+)
+
+type PostSESKeys struct {
+	AccessKey string `form:"access_key" validate:"required,alphanum"`
+	SecretKey string `form:"secret_key" validate:"required"`
+	Region    string `form:"region" validate:"required"`
+}
+
+func (p *PostSESKeys) trimSpaces() {
+	p.AccessKey = strings.TrimSpace(p.AccessKey)
+	p.SecretKey = strings.TrimSpace(p.SecretKey)
+	p.Region = strings.TrimSpace(p.Region)
+}
+
+func (p *PostSESKeys) Validate(v *validator.Validate) error {
```

Comment:
@Gligor23 since we're using an interface, this should be a generic function, something like:

func Validate(body RequestBody, v *validator.Validate) error {
  body.trimSpaces()

...
}